packages/eva/themes/dark.json

```diff
@@ -11,6 +11,20 @@
   "color-basic-1000": "#151A30",
   "color-basic-1100": "#101426",
 
+  "color-basic-100-transparent-8": "rgba(255, 255, 255, 0.08)",
```

Comment:
I guess it makes sense to define transparent colors for **any** basic/semantic color variable and give it more generic names like `color-basic-100-level-1`, `color-basic-100-level-2`, etc

Reply:
Make sense. But on the other hand, a potential confusion is possibly related to the status (level-1, -2, etc.) of the Layout component.